agent/agent.go

```diff
@@ -264,7 +263,8 @@ func initializeStateManager(cfg *config.Config, taskEngine engine.TaskEngine, cl
 		statemanager.AddSaveable("ContainerInstanceArn", containerInstanceArn),
 		statemanager.AddSaveable("Cluster", cluster),
 		statemanager.AddSaveable("EC2InstanceID", savedInstanceID),
-		statemanager.AddSaveable("ACSSeqNum", sequenceNumber),
+		//not tracking ACSSeqNum for now
```

Comment:
What does "not tracking" mean here?  If `ACSSeqNum` is only preserved to avoid changing the state file version, can you rephrase to indicate that?